.github/boring-cyborg.yml

```diff
@@ -344,6 +344,27 @@ labelPRBasedOnFilePath:
     - airflow-core/docs/ui.rst
     - airflow-core/src/airflow/ui/**/*
 
+  area:translations:
+    - airflow-core/src/airflow/ui/src/i18n/**/*
+
+  translation:default:
+    - airflow-core/src/airflow/ui/src/i18n/locales/en/*
+
+  translation:de:
```

Comment:
Do we need label per language? Seems a bit excessive.

Reply:
Per language seems to make sense to me as the reviewers of one language might not be able to review another language.

Reply:
Translations is under havey development but I expect it to be reduced soon. At the end translation is 2 files which will remain stable with little to no changes (unlike provider packages).

Having a label per translation mean that someone needs to maintain it, creating the label, updating the labeling bot etc...

If we want label per language we need someone to volunteer to own it, including marking issues and writing the triage policy for it. This cant be left to the owner of a specific language.

To me it seems like a lot of work for little value but if someone wants to have it and willing to set it up feel free

Reply:
In the long term, if we maintain translations as sort of "mini Python packages" - I think that it makes sense to treat them like providers, so each will have its own label. Issues/PRs will be easier to filter for both analytical and practical purposes, not only by code owners.
I agree though that it needs to be maintained or automated (thanks Jens!).